apis/definition.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: xeks.aws.platform.upbound.io
 spec:
   connectionSecretKeys:
-    - kubeconfig
```

Comment:
Can we not drop this one please, we use it down the line and will just limit flexibility

Reply:
updated the commit to keep it, many thanks for the review 🙇

Reply:
My pleasure :)